receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -96,4 +98,4 @@ metrics:
     unit: 1
     gauge:
       value_type: int
-    attributes: [group, topic]
+    attributes: [group, topic]
```

Comment:
Nit: unnecessary removal of the training line. Usually, linters complain about not having them

Reply:
Understand. 
This stems from an IDE, I think I should not enable autoformatting.